src/Tizen.NUI/src/internal/WebView/WebView.cs

```diff
@@ -662,6 +749,86 @@ public bool KeyEventsEnabled
             }
         }
 
+        /// <summary>
+        /// Background color of web view.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public new Color BackgroundColor
```

Comment:
Original `BackgroundColor` is a background visual's color. Background visual can have larger size than a view.  And the property is tightly connect to a `Background` and a `BackgroundImage` properties. So overriding original logic has a lot of potential problem.
Plus, it is not good that a property and a bindable property have different name: BackgroundColor and WebViewBackgroundColor.

I think setting WebView's background should be done in DALi.
(When a user sets the BackgroundColor property, a WebView control should set both actor's background color and ewk's background color).
But here we have no time to update DALi again, I think `ContentBackgroundColor` is enough for now.

Reply:
Done.

Reply:
Please make sure the name is not using `Document` unless it really means the document's background color. I assume `ContentBackgroundColor` is enough as I commented.

Reply:
Done.